blueprint/core/blueprint_GenericEditor.cpp

```diff
@@ -21,14 +21,14 @@ namespace blueprint
         for (auto& p : processor->getParameters())
             p->addListener(this);
 
-        // Now we can provision the app root
-        assignNewAppRoot(code);
+        // Setup the ReactApplicationRoot callbacks and evaluate the supplied JS code/bundle
+        registerAppRootCallbacks();
+        appRoot.evaluate(code);
 
-        // If an error showed up, our appRoot is already gone
-        if (appRoot)
-            addAndMakeVisible(appRoot.get());
+        // Add ReactApplicationRoot as child component
+        addAndMakeVisible(appRoot);
 
-        // Set an arbitrary size, should be overriden from outside the constructor
+        // Set an arbitrary size, should be overridden from outside the constructor
```

Comment:
:)

Reply:
Ha Jetbrains IDE's do yellow squiggly thinks for spell correct. It's actually a bit annoying ...